### PR TITLE
MDCT-223: Fix Dead Spots in Header Menu

### PR DESCRIPTION
**Pull request by karla-vm**
# Description

Closes [MDCT-223](https://qmacbis.atlassian.net/browse/MDCT-223). Users have trouble finding the clickable area of the CARTS header drop-down menu. Now it should be clickable all-around. Also, changed **Helpdesk** to **Contact Us** based on Design's suggestion.

## How to test

`cd frontend`
`docker-compose -f docker-compose.local.yml down -v --rmi local`
`docker-compose -f docker-compose.local.yml up -d`

Navigate to `localhost:81` and log in as any user type. On the top-right, verify that the menu items are clickable. 

## Dependencies

N/A

# Get it done

Clicks!

## Code authors checklist

- [x] I have performed a self-review of my code
- [ ] I have added thorough tests. [Testing Doc](https://qmacbis.atlassian.net/wiki/spaces/CM/pages/2914025525/Test+Suite+and+Testing+Research)
- [x] Necessary analytics were added, or no new analytics were required
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned the PR to myself

## Reviewers Checklist (Two different people)

- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

## Assignee

- [ ] I have closed the PR after the review and necessary changes (squashing preferred)

### CODE REVIEW

**Review by juliuscaesar — APPROVED** [review]
Looks good to me locally!

File: frontend/react/src/components/layout/Logout.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import React from "react";
 import { useOktaAuth } from "@okta/okta-react";
 import { Button } from "@cmsgov/design-system-core";
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```
Comment (BearHanded) [inline frontend/react/src/components/layout/Logout.js:4]: If we're not using these, can we remove the comments?

File: frontend/react/src/components/layout/Logout.js [hunk 2]
```diff
@@ -29,12 +29,10 @@ const Logout = () => {
   return (
     <Button
       type="button"
-      inversed
       variation="transparent"
       onClick={loginInfo.indexOf(localUserPrefix) >= 0 ? localLogout : logout}
     >
-      <FontAwesomeIcon icon={faSignOutAlt} />
```
Comment (BearHanded) [inline frontend/react/src/components/layout/Logout.js:36]: I don't see that the ticket requests these icons removed, can you confirm that?

File: frontend/react/src/components/layout/Logout.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import React from "react";
 import { useOktaAuth } from "@okta/okta-react";
 import { Button } from "@cmsgov/design-system-core";
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faSignOutAlt } from "@fortawesome/free-solid-svg-icons";
+// import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```
Comment (BearHanded) [inline frontend/react/src/components/layout/Logout.js:4]: If we're not using these, can we remove the comments?

**Review by BearHanded — APPROVED** [review]

**Pull request merged**